speedtest.go

```diff
@@ -79,11 +78,15 @@ func (adm *AdminClient) Speedtest(ctx context.Context, opts SpeedtestOpts) (Spee
 	if resp.StatusCode != http.StatusOK {
 		return SpeedTestResult{}, httpRespToErrorResponse(resp)
 	}
-	respBytes, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		return SpeedTestResult{}, err
+	for {
+		var result SpeedTestResult
```

Comment:
```suggestion
		var result SpeedTestResult
```

This needs to be out of `for {`